MinimedKit/Models/PumpColor.swift

```diff
@@ -7,8 +7,9 @@
 
 
 public enum PumpColor: String {
-    case blue   = "B"
```

Comment:
The intent is for this to be entered based on the letter on the back of the pump. The rest of the design (highlighting the location of those 3 letters visually) and text (describing that the region and color are encoded in those three letters) is built around this.

Reply:
I think the letter might have been 'H'?

Reply:
And for the image, it looks like the inner sticker part is lighter than the outer plastic casing:
![sany0046](https://user-images.githubusercontent.com/14649/45557151-93910400-b802-11e8-99b2-1256d10e055f.jpg)